### PR TITLE
Feature/prime 2145 site pagination

### PR DESCRIPTION
**Pull request by hogmann**
Another flavour of the pagination, this time on sites. A lot of the same testing either add test data to exceed 100+ sites or adjust pageSize in PaginatedList in the backend.

### CODE REVIEW

**Comment by AMiKaH** [comment]
LGTM

**Comment by neophyte57** [comment]
Seeing this warning
image
related to 
image
?  

Not part of this PR but perhaps should be addressed

**Comment by neophyte57** [comment]
Organizations and their sites don't seem to be grouped correctly.  `develop` code:

image

`feature/PRIME-2145-site-pagination` code:

image
image

**Comment by hogmann** [comment]
> Seeing this warning image related to image ?
> 
> Not part of this PR but perhaps should be addressed

Both your comments stem from the same thing and relate, good catch. I missed putting a OrderBy in the site service method. So by adding an orderby displayId that should sort all the orgs together and get rid of that warning 👍

**Review by johnnyl6 — APPROVED** [review]
LGTM

File: prime-dotnet-webapi/Services/CommunitySiteService.cs [hunk 1]
```diff
@@ -45,6 +46,31 @@ public async Task<IEnumerable<CommunitySite>> GetSitesAsync(int? organizationId
...
+                    .Search(
+                        s => s.DoingBusinessAs,
+                        s => s.PEC,
+                        s => s.Organization.Name,
+                        s => s.Organization.DisplayId.ToString(),
+                        s => s.Organization.SigningAuthority.FirstName + " " + s.Organization.SigningAuthority.LastName)
+                    .Containing(searchOptions.TextSearch)
+                )
+                .ProjectTo<CommunitySiteAdminListViewModel>(_mapper.ConfigurationProvider)
+                .OrderBy(s => s.DisplayId)
```
Comment (neophyte57) [inline prime-dotnet-webapi/Services/CommunitySiteService.cs:68]: `OrderBy` helped but still Organization should come before its Sites

image

Comment (neophyte57) [reply prime-dotnet-webapi/Services/CommunitySiteService.cs:68]: OK, never mind about comment re: "Organization should come before its Sites" but the Sites within an Org should be sorted more intuitively, by Submission Date or whatever logic was implemented before?

**Pull request merged**